INFO     root:conftest.py:226 === Starting test_negative_get_transactions_invalid_coin Test (Negative) ===
INFO     root:conftest.py:227 API Response: 404 | 
INFO     root:conftest.py:237 Test 'test_negative_get_transactions_invalid_coin' (Negative) SUCCESS.
INFO     root:conftest.py:238 === Ending test_negative_get_transactions_invalid_coin Test (Negative) ===